{"uuid":"6fbf55f9-dbda-4d47-b053-f46415f24f32","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["e56e476b-fec7-4fe7-9746-ecaa792db723"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"c4a111d5-927b-444f-8454-85712e112162-attachment","type":"image/png"},{"name":"Myscreenshot","source":"45e8409f-b676-478b-bfde-5ae0d7dd4a81-attachment"}],"parameters":[],"start":1696887709498,"stop":1696887709681},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887709682,"stop":1696887709682},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887709682,"stop":1696887709683},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887709683,"stop":1696887709683}],"start":1696887709494,"stop":1696887709684}